db/seeds.rb

```diff
@@ -3,5 +3,9 @@
 #
 # Examples:
 #
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
+  seed_user = User.create({
+    name: 'seed_user',
+    email: 'test_user@email.com',
+    password: 'secret!'
+    })
```

Comment:
this is a nitpick, but indentation we'd use would look something like this:
```ruby
seed_user = User.create({
  name: 'seed_user',
  email: 'test_user@email.com',
  password: 'secret!'
})
```

Reply:
Cool - thanks for the feedback!